packages/app/src/app/components/CodeEditor/Monaco/workers/fetch-dependency-typings.js

```diff
@@ -173,15 +173,19 @@ function fetchFromMeta(dependency, version, fetchedPaths) {
   return doFetch(depUrl)
     .then(response => JSON.parse(response))
     .then(meta => {
-      const filterAndFlatten = files =>
+      const filterAndFlatten = (files, filter) =>
         files.reduce((paths, file) => {
-          if (/\.d\.ts$/.test(file.name)) {
+          if (filter.test(file.name)) {
             paths.push(file.name);
           }
           return paths;
         }, []);
 
-      const dtsFiles = filterAndFlatten(meta.files);
+      let dtsFiles = filterAndFlatten(meta.files, /\.d\.ts$/);
+      if (dtsFiles.length === 0) {
+        // if no .d.ts files found, fallback to .ts files
```

Comment:
@CompuIves Is this safe?

Reply:
What would the possible attack vector be here?

Reply:
I meant "safe" as could it break any existing sandbox, not security related. :slightly_smiling_face:

Reply:
Oh, as safe as a chicken in a barn! 🐔